Makefile

```diff
@@ -66,21 +63,19 @@ copy_static_prebuilt:
 .PHONY: build
 build:
 	@echo -e "\033[0;32mBUILD:\033[0m"
-	yarn lint-markdown
+	$(MAKE) lint
 	./scripts/build-site.sh
 
-.PHONY: pulumify
-pulumify:
-	@echo -e "\033[0;32mBUILD PULUMIFY:\033[0m"
-	$(MAKE) ensure
-	./scripts/build-site.sh preview
+.PHONY: lint
+lint:
+	yarn run lint-markdown
 
 .PHONY: test
 test:
 	$(MAKE) check_links_local
 
 .PHONY: check_links_local
-check_links_local::
+check_links_local:
```

Comment:
I am no `make` expert, but isn't the second `:` required here and the few other places below where they were removed?

Reply:
Definitely not required, no. There does seem to be _some_ use for them, but [even the GNU Make docs](https://www.gnu.org/software/make/manual/html_node/Double_002dColon.html) point out that those uses are quite specific and rare:

> Double-colon rules are somewhat obscure and not often very useful; they provide a mechanism for cases in which the method used to update a target differs depending on which prerequisite files caused the update, and such cases are rare. 

In our case, since we're just using Make targets as task wrappers (as opposed using to compile software binaries, for example), they aren't really doing anything, so I figured I'd remove them.